src/commands/mock.ts

```diff
@@ -47,5 +47,31 @@ export const Mock = {
         }
         mockTargets.add(target.id);
         await interaction.reply(`Sure thing, I will begin mocking ${target}!`);
+    },
+
+    Effect: async (message: Message): Promise<void> => {
+
+        let shouldBeLower = true;
+        let length = message.content.length;
+        let userMessage = message.content
+        let mockMessageArray = [...userMessage];
+
+        for (let i = 0; i < length; i++) {
+            if (userMessage[i].toUpperCase() !== userMessage[i].toLowerCase()) {
```

Comment:
Out of curiosity, any reason for having this if statement? Usually mocking can just be alternating cases, but it's also up to you